src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -208,6 +209,8 @@ public Thickness BorderThickness
             }
         }
 
+        public double PrimaryScreenRenderScaling => Screen.AllScreens.FirstOrDefault(screen => screen.Primary)?.PixelDensity ?? 1;
```

Comment:
this can be private I think?

Reply:
@danwalmsley yes, although it is private property then. Changed.